src/codestyle.sh

```diff
@@ -73,6 +73,49 @@ function codestyle_main()
   checkpatch=$(join_path "$kernel_root" 'scripts/checkpatch.pl')
   cmd_script="perl ${checkpatch} ${checkpatch_options}"
 
+  [[ -n "${options_values['START_LINE']}" ]] && start_line=${options_values['START_LINE']}
+  [[ -n "${options_values['END_LINE']}" ]] && end_line=${options_values['END_LINE']}
+
```

Comment:
All the start_line and end_line manipulation should be encapsulated in its own functions.